traj/scripts/traj_segment_example

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+"""
+Simple example that fits general trajectory segment, given the start and
+end positions/velocities. Start and end acceleration/jerk are assumed to be zero.
+"""
+from matplotlib import pyplot as plt
+import traj
+
+
+def test_seven_segment(p_start, p_end, v_start, v_end, p_max, v_max, a_max, j_max):
+    position, velocity, acceleration, jerk = traj.fit_traj_segment(p_start, p_end, v_start, v_end, p_max, v_max, a_max, j_max)
+    traj.plot.plot_trajectory(plt, position, velocity, acceleration, jerk, n_points=100, v_max=v_max, a_max=a_max, j_max=j_max)
+    plt.show()
+    
+p_max=30
+v_max=3.0
+a_max=4.0
+j_max=10.0   
+
+############### CASE A: v_start = v_end = 0    
+#case A1:  no limit is reached     
+test_seven_segment(0.0, 1.0,      0.0, 0.0,     p_max, v_max, a_max, j_max)    
+#case A2:  acc_limit is reached     
+test_seven_segment(0.0, 3.0,      0.0, 0.0,     p_max, v_max, a_max, j_max)    
+#case A3:  vel_limit and acc_limit are reached    
+test_seven_segment(0.0, 5.0,      0.0, 0.0,     p_max, v_max, a_max, j_max)    
+
+
+############## CASE B: v_start = v_end !=0
+#case B1:  no limit is reached
+test_seven_segment(0.0, 2.0,      1.0, 1.0,     p_max, v_max, a_max, j_max)    
+#case B2:  vel_limit is reached 
+test_seven_segment(0.0, 3.0,      2.5, 2.5,     p_max, v_max, a_max, j_max)    
+#case B3:  acc_limit is reached
+test_seven_segment(0.0, 3.0,      0.5, 0.5,     p_max, v_max, a_max, j_max)    
+#case B4:  vel_limit and acc_limit are reached   
+test_seven_segment(0.0, 10.0,     0.5, 0.5,     p_max, v_max, a_max, j_max)    
+
+
+
+############## CASE C: v_start != v_end !=0, and v_start < v_end [acceleration]
+#case C1:  no limit is reached
+test_seven_segment(0.0, 2.0,       0.5, 1.5,    p_max, v_max, a_max, j_max)    
+#case C2:  vel_limit is reached 
+test_seven_segment(0.0, 5.0,      2.0, 2.5,     p_max, v_max, a_max, j_max)    
+#case C3:  acc_limit is reached
+test_seven_segment(0.0, 3.0,      0.5, 2.5,     p_max, v_max, a_max, j_max)    
+#case C4:  vel_limit and acc_limit are reached   
+test_seven_segment(0.0, 5.0,       0.5, 2.5,    p_max, v_max, a_max, j_max)    
```

Comment:
The test on line 49 appears to generate a trajectory with 8 segments instead of 7:
![8segment](https://user-images.githubusercontent.com/1538056/73425466-8df86900-42e6-11ea-933c-96aa21c775ab.png)
Is that expected?

Reply:
yes, and some cases with complex motion (where v_start and v_end are +/- or -/+)  will have 9 segments. I added this to guarantee that starting and ending acceleration is zero.